demisto_sdk/__main__.py

```diff
@@ -1971,8 +1977,10 @@ def generate_docs(ctx, **kwargs):
         return 1
 
     if input_path.is_file():
-        if input_path.suffix.lower() != ".yml":
-            logger.info(f"[red]input {input_path} is not a valid yml file.[/red]")
+        if input_path.suffix.lower() not in {".yml", ".md"}:
+            logger.info(
```

Comment:
```suggestion
            logger.error(
```
Maybe raise an exception?

Reply:
There are some other lines before and after that use `logger.info` and `return 1` instead of `logger.error` and raising an exception. Do you want to change them as well?

Reply:
Please, yes
The behavior should be raising an exception with a matching error message, then `logger.error` (once) when catching it.